README.rst

```diff
@@ -53,9 +53,11 @@ Official documentation for MLflow can be found at https://mlflow.org/docs/latest
 
 Community
 ---------
-To discuss MLflow or get help, please subscribe to our mailing list (mlflow-users@googlegroups.com) or
+To discuss MLflow, please subscribe to our mailing list (mlflow-users@googlegroups.com) or
```

Comment:
Could we list more specific channels of communication first & make the mailing list the fall-through, e.g.:

```
For help or questions about MLflow usage (e.g. "how do I do X?") see the `docs <https://mlflow.org/docs/latest/index.html>`_  or `Stack Overflow <https://stackoverflow.com/questions/tagged/mlflow>`_

To report a bug, file a documentation issue, or file a feature request, please open a GitHub issue.

For release announcements and other discussions, please subscribe to our mailing list (mlflow-users@googlegroups.com) or join us on Slack at https://tinyurl.com/mlflow-slack.
```